### PR TITLE
I need help with seed file. Im starting to think that its an issue wi…

### PR DESCRIPTION
**Pull request by djimenez017**
I need help with seed file. Im starting to think that its an issue with the ways that i connected the tables, Could you check and see if my suspicion is correct? Or is it just syntax error?

### CODE REVIEW

File: db/migrate/20230316050555_create_addresses.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def change
       t.text :zip_code, null: false
       t.text :state, null: false
       t.text :country, null: false
-      t.references :yard_sale, foreign_key: true, null: false
+      t.references :yard_sales, foreign_key: true, null: false
```
Comment (kervin5) [inline db/migrate/20230316050555_create_addresses.rb]: I think this should stay as “yard_sale” especially because this migration was already applied. Changing this doesn't really have any effect unless you rollback. Regardless, this should stay as `yard_sale`

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,80 @@
...
+)
+user3.yard_sales.create(
+  title: "Monday",
+  description: "Lots of cool stuff",
+  date: "Tue, 08 Dec 2015",
+  hours: "Sat, 01 Jan 2000 12:26:40.000000000 UTC +00:00"
+)
+
+# create some addresses for each yardsale
+user1.yard_sales.addresses.create(
```
Comment (kervin5) [inline db/seeds.rb]: yard_sales.address is not a valid method because a yard sale can only have one address and yard_sales returns an array of all yard_sales associated to that user so you would have to iterate over each record or use ".first"

**Comment by djimenez017** [comment]
can you view what i currenlty have? maybe some feedback? Also i need help with the js for the hamburger menu. the Side drawer is not showing. Any tips?

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -1,4 +1,8 @@
 class PagesController < ApplicationController
   def home
+    @yardsales = YardSale.all
```
Comment (kervin5) [inline app/controllers/pages_controller.rb:3]: Cool 👍🏾

File: app/controllers/yard_sales_controller.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class YardSalesController < ApplicationController
+  def show
+    @yardsale = YardSale.find(params[:id])
```
Comment (kervin5) [inline app/controllers/yard_sales_controller.rb]: Looks good. I doesn't have to be addressed in this PR, but it may be a good idea to handle ids that are invalid or don't correspond with any records. 404 maybe.

File: app/helpers/yard_sales_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module YardSalesHelper
+end
```
Comment (kervin5) [inline app/helpers/yard_sales_helper.rb]: Is this file needed?

File: app/javascript/application.js [hunk 6]
```diff
@@ -1,3 +1,27 @@
...
-import "controllers"
+import "@hotwired/turbo-rails";
+import "controllers";
+
+document.addEventListener("DOMContentLoaded", () => {
+  // Get all "navbar-burger" elements
+  const $navbarBurgers = Array.prototype.slice.call(
+    document.querySelectorAll(".navbar-burger"),
+    0
+  );
```
Comment (kervin5) [inline app/javascript/application.js:10]: I know that you got this from Bulmas website, but I wonder why they used weird stuff live $ for the name of the variables or things like this instead of just doing:

```javascript
const $navbarBurgers = [...document.querySelectorAll('.navbar-burger')];
```
or even just

```
const $navbarBurgers = document.querySelectorAll('.navbar-burger');
```

Probably their example is just targeting ES5.

File: app/views/pages/home.html.erb [hunk 7]
```diff
@@ -8,74 +8,45 @@
...
...B6rVX9RYdpxS4fxUVxkqpKjwGd6q3HxUylMqfgXeesrE2w6LxCmQCToB7t8P/OJQ/JjjOT3Y0hTbM8Vy/8AWUaWcpj79I6KnzWIPsqn3xwedMrXSpQqs+WqOjIVrqVI1AiZ2G/eMOp85kb+mKPEcjSrLpq0kqL5OoP4Tt8sLJfUFATkHia1cqlORroAUXCmRKdKkEWKsoBB2N/LCb9OB6sr7VP+zBfiuTo8MqLmKB0gnTXy+uS1NvvKCS0ob7x7XlT+lHjlLNNQNINpUP1NHVq0wRee3fGkV+VifKNF5UqxlMr/AJKfsj+GLPG1rvRYZeoadUXFh1Rup1AgT59jHacUeWP/AImW/wAlP2Rgqu/8+QxEujXDFajEk6pDSZmxBm8jsZ7YrVgRddx/MfhbDn9IuVRcyhCgF6QZj5kGJPrAjCtjVPVkNEFFpHf5jEyqcSDEgwwOFXHYp47XEowgIPDxIq4kx0MAFHOi9P8AxjFniEeE/t+/EGd3p/5i/rhoXhlJuHZisUmoq2aTbqHaY/LCsAHw+ixgAEuQBpAMgAWEb+vvg7k+V8zU/wBloHnUOn8vi/LD1ylTC5TKwANVBGPqSgMnzM3wVbEOWykhOynIg/21Yn0Rf+5p/ZwWp8p5ZRHhlv8AE7z+IIj2j8MGv5/PHwxDky6RU4bwihSLGjT0k7yWJj5k29sXDGPjvjtLwcIDnHjHHzbY5O+AD0+mPRMWie3vjnscdDt/PngAC8so+Wy1KjWRpQEF6YLqZYm2nrG+5X54I1k8ZSaVSm67TvB7yymJHlAxcfEVbIU3Oor1afjUlW9tSkNHpMYLATee8q65ZVKqwAYkGCNzB8xjLqpCmR8x+n8PwxsnDs21bKsasOdZXqVSIFtojtvvjKs9QXSx0gbmwgb9gLD5Y2gyGqKcOTb+fxxZo5IsbsZ8/wDycS5O6j2GGHlTLJUrhXUMPI4bdAtnOV5brqodUaReLSfIr/a2/TfDbwebmilWnU+8oUi/rPRf+9+WC/BMlT0kaQAJhRZb+aix+YwZS0KLACwFgPkMYN2VVAunw3UoNRVR5maRK+xI+Fm33BGFDmKhxNZHiPUpjY0YBI/vBAHnz3GH848ff8MNOgZhDNBM7zf37zhd4sNJCfdBJX0DQY+Rx+heIcJo5gkVqavbfZv+YQ354/PnHbVmHYE42i7M5Kj/2Q=="
```
Comment (kervin5) [inline app/views/pages/home.html.erb:20]: Storing dynamic images is always tricky. Ideally you want to use a specialized service such as s3 or cloudinary, but you could also use active storage to save them in your db: https://levelup.gitconnected.com/rails-image-upload-101-f9bf245e389b.

File: app/views/pages/home.html.erb [hunk 8]
```diff
@@ -8,74 +8,45 @@
...
-            <span class="tag is-primary">Collectibles</span>
-            <span class="tag is-primary">Clothing</span>
-            <span class="tag is-primary">Electronics</span>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+<% @yardsales.each do |yardsale| %>
```
Comment (kervin5) [inline app/views/pages/home.html.erb]: It may be good to have a condition to show "No yardsales available" or something like that if `@yardsales.length` == 0

File: app/views/pages/home.html.erb [hunk 9]
```diff
@@ -8,74 +8,45 @@
...
-            <span class="tag is-primary">Clothing</span>
-            <span class="tag is-primary">Electronics</span>
+          <div class="content">
+            <%= truncate(yardsale.description, length:200) %>
+            <br />
+            <br />
+            <div>
+              <span class="tag is-primary">Collectibles</span>
+              <span class="tag is-primary">Clothing</span>
+              <span class="tag is-primary">Electronics</span>
```
Comment (kervin5) [inline app/views/pages/home.html.erb]: Is this hardcode for testing purposes?

**Review by kervin5 — APPROVED** [review]
A couple of questions, but no blockers.

File: app/helpers/yard_sales_helper.rb (hunk 5 above)
Comment (djimenez017) [reply app/helpers/yard_sales_helper.rb]: No, i dont think its needed. Removed

File: app/javascript/application.js (hunk 6 above)
Comment (djimenez017) [reply app/javascript/application.js:10]: yeah i was wondering that as well. Do you think i should refactor? or leave as is?

File: app/views/pages/home.html.erb (hunk 7 above)
Comment (djimenez017) [reply app/views/pages/home.html.erb:20]: Thank you, I will do this. After installing it seems like it just a matter of adding to the image model.

File: app/views/pages/home.html.erb (hunk 9 above)
Comment (djimenez017) [reply app/views/pages/home.html.erb]: Yes, i will include tags in the seed file

File: app/controllers/yard_sales_controller.rb (hunk 4 above)
Comment (djimenez017) [reply app/controllers/yard_sales_controller.rb]: added a check if the id doest not exists. Redirects to 404

File: app/views/pages/home.html.erb (hunk 8 above)
Comment (djimenez017) [reply app/views/pages/home.html.erb]: used @yardsales.any? its sooo cool and practical lol

File: app/views/pages/home.html.erb (hunk 9 above)
Comment (djimenez017) [reply app/views/pages/home.html.erb]: tags included in the seedfile. Now mapping though the tags
